### DISCUSSION TITLE
Editor / Remote for Lexicon MPX1

### DISCUSSION THREAD

**Discussion by Tonfisch** [RomanKubiak/ctrlr#562] [Show and tell]
Hello,
Lexicon MPX1 is a wonderfull sounding device, but it breaks fingers if you try to edit  something.
So i tried to help with that.
It´s my second beast. Has taken some time.

https://github.com/Tonfisch/MPX1

![MPX1_1](https://github.com/RomanKubiak/ctrlr/assets/115865850/e7e54dc9-7247-439c-a706-2c4286554b4b)

![MPX1_2](https://github.com/RomanKubiak/ctrlr/assets/115865850/fe45a58d-a5bd-4d61-b245-18d8e60b31af)

Have Fun

Tonfisch

**Comment by dobo365** [comment]
Thx for that one. I was also planning to do one (but not immediately as I have still a lot of work on my other panels) as I just acquired an brand "new" MPX1.
My initial idea was to make one picture that would fit all algorithms then to highlight the connections on it and allow users to edit the params acc to the selected elements of the FX.
As you have done the thought work of the Midi implementation, I'll see what I can do to improve the panel in general.
The knob is just one among other available in the jnokbman gallery ;-)
Thx again!

**Reply by Tonfisch** [reply]
that would be great. Btw. i learned much from your code.
Thx for that!
